# <span style="font-size:20pt;font-family:Montserrat, sans-serif;color:rgb(92, 160, 250);font-weight:700;vertical-align:baseline;white-space:pre-wrap;">Graded Assignment 3: 9 to 5 - K. Jones</span>

<span style="font-size:10pt;font-family:&quot;Times New Roman&quot;;color:rgb(102, 102, 102);vertical-align:baseline;white-space:pre-wrap;">Time to show off your SQL skills! For each question, copy the SQL query you used and make note of the answer.</span>

## <span style="font-size:12pt;font-family:Montserrat, sans-serif;color:rgb(53, 74, 95);font-weight:700;vertical-align:baseline;white-space:pre-wrap;">The Dataset</span>

<span style="font-size:10pt;font-family:&quot;Times New Roman&quot;;color:rgb(102, 102, 102);vertical-align:baseline;white-space:pre-wrap;">For this assignment, you will be using the Bureau of Labor Statistics (BLS) Current Employment Survey (CES) results which can be found on</span> [<span style="font-size:10pt;font-family:&quot;Times New Roman&quot;;color:rgb(17, 85, 204);vertical-align:baseline;white-space:pre-wrap;">Kaggle</span>](https://www.kaggle.com/bls/employment)<span style="font-size:10pt;font-family:&quot;Times New Roman&quot;;color:rgb(102, 102, 102);vertical-align:baseline;white-space:pre-wrap;">.</span>

## <span style="font-size:12pt;font-family:Montserrat, sans-serif;color:rgb(53, 74, 95);font-weight:700;vertical-align:baseline;white-space:pre-wrap;">Business Issue</span>

<span style="font-size:10pt;font-family:&quot;Times New Roman&quot;;color:rgb(102, 102, 102);vertical-align:baseline;white-space:pre-wrap;">You are working for the Bureau of Labor Statistics with the United States government and have been approached by your boss with an important meeting request. You have been asked by your supervisor to meet with Dolly Parton whose nonprofit is looking to shed light on the state of employment in the United States. As part of the 9 to 5 project, their research is focused on production and nonsupervisory employees and how those employees fare compared to all employees in the United States. While the data the BLS collects from the CES is publicly available, Dolly Parton and her colleagues need your assistance navigating the thousands of rows in each table in LaborStatisticsDB.</span>

## <span style="font-size:12pt;font-family:Montserrat, sans-serif;color:rgb(53, 74, 95);font-weight:700;vertical-align:baseline;white-space:pre-wrap;">About the Dataset</span>

<span style="font-size:10pt;font-family:&quot;Times New Roman&quot;;color:rgb(102, 102, 102);vertical-align:baseline;white-space:pre-wrap;">This dataset comes directly from the Bureau of Labor Statistics’ Current Employment Survey (CES). Here are some things you need to know:</span>

1. <span style="font-size:10pt;vertical-align:baseline;white-space:pre-wrap;">The industry table contains an NAICS code. This is different from the industry code. NAICS stands for North American Industry Classification System.</span>
    
2. <span style="font-size:10pt;vertical-align:baseline;white-space:pre-wrap;">Series ID is composed of multiple different codes. CES stands for Current Employment Survey, the name of the survey which collected the data. The industry code as specified by the BLS and the data type code as specified in the datatype table.</span>
    

## <span style="font-size:12pt;font-family:Montserrat, sans-serif;color:rgb(53, 74, 95);font-weight:700;vertical-align:baseline;white-space:pre-wrap;">Set Up</span>

<span style="font-size:10pt;font-family:&quot;Times New Roman&quot;;color:rgb(102, 102, 102);vertical-align:baseline;white-space:pre-wrap;">To connect to the database, use the same connection info used during the SQL lessons.&nbsp;</span>  

<span style="font-size:10pt;font-family:&quot;Times New Roman&quot;;color:rgb(102, 102, 102);vertical-align:baseline;white-space:pre-wrap;">For the assignment, we will be using the LaborStatisticsDB database.</span>

## <span style="font-size:12pt;font-family:Montserrat, sans-serif;color:rgb(53, 74, 95);font-weight:700;vertical-align:baseline;white-space:pre-wrap;">Database Exploration</span>

<span style="font-size:10pt;font-family:&quot;Times New Roman&quot;;color:rgb(102, 102, 102);vertical-align:baseline;white-space:pre-wrap;">To start with, let’s get to know the database further.</span>

1. <span style=;color:rgb(102, 102, 102);font-family:&quot;Times New Roman&quot;;font-size:10pt;white-space:pre-wrap;">Use this space to make note of each table in the database, the columns within each table, each column’s data type, and how the tables are connected. You can write this down or draw a diagram. Whatever method helps you get an understanding of what is going on with LaborStatisticsDB.</span>

To add a photo, diagram or document to your file, drop the file into the folder that holds this notebook.  Use the the link button to the right of the  \</\> symbol in the gray part of this cell, the link is just the name of your file.

  

[Labor Database Diagram](vscode-file://vscode-app/c:/Users/kalaj/AppData/Local/Programs/Azure%20Data%20Studio/resources/app/out/vs/code/electron-browser/workbench/Labor%20Database%20Diagram)

2. <span style=" color: rgb(102, 102, 102); font-family: Arvo, serif; font-size: 10pt; white-space: pre-wrap;">What is the datatype for women employees?</span>

<span style="font-size: 13.3333px; white-space: pre-wrap;">This question is a bit vauge. There are two options/data types with "Women" in the title. Both options are technicially "nvarchar" data types. However the Data Codes are <u>10 for "Women Employees"</u> and <u>39 for "Women Employees-to-all Employees Ratio"</u></span>

In [2]:
USE LaborStatisticsDB

SELECT *
FROM datatype
WHERE data_type_text LIKE '%women%'

(2 rows affected)

Total execution time: 00:00:00.127

data_type_code data_type_text 10 WOMEN EMPLOYEES 39 WOMEN EMPLOYEES-TO-ALL EMPLOYEES RATIO

3. <span style=" font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">What is the series id for&nbsp; women employees in the commercial banking industry in the financial activities supersector?</span>

<span style="font-size: 13.3333px; white-space: pre-wrap;"><b><u>The Series Id is CES5552211010</u></b></span>

In [3]:
SELECT industry_name, industry_code
FROM industry
WHERE industry_name = 'commercial banking'

SELECT supersector_name, supersector_code
FROM supersector
WHERE 
supersector_name = 'financial activities'

SELECT series_id
FROM LaborStatisticsDB.dbo.series
WHERE data_type_code = 10 AND supersector_code = 55 AND industry_code = 55522110;

(1 row affected)

(1 row affected)

(2 rows affected)

Total execution time: 00:00:00.086

industry_name industry_code Commercial banking 55522110

supersector_name supersector_code Financial activities 55

series_id CES5552211010 CEU5552211010

## <span style="font-size: 12pt; font-family: Montserrat, sans-serif; color: rgb(53, 74, 95); font-weight: 700; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Join in on the Fun</span>

<span style="font-size: 10pt; font-family: Arvo, serif; color: rgb(102, 102, 102); font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Time to start joining! You can choose the type of join you use, just make sure to make a&nbsp; note!</span>

1. <span style=" font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">Join&nbsp; annual_2016 with series on series_id. We only want the data in the annual_2016 table to be included in the result.</span>

In [16]:
SELECT TOP 10 *
FROM annual_2016 AS a
LEFT JOIN series AS s
ON a.series_id = s.series_id;

(10 rows affected)

Total execution time: 00:00:00.249

id series_id year period value footnote_codes original_file series_id supersector_code industry_code data_type_code seasonal series_title 0 CEU5500000007 2016 M13 36.9 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000007 55 55000000 7 U Average weekly hours of production and nonsupervisory employees 1 CEU5500000008 2016 M13 26.11 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000008 55 55000000 8 U Average hourly earnings of production and nonsupervisory employees 2 CEU5500000030 2016 M13 962.73 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000030 55 55000000 30 U Average weekly earnings of production and nonsupervisory employees 3 CEU5500000031 2016 M13 411.29 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000031 55 55000000 31 U Average weekly earnings of production and nonsupervisory employees 4 CEU5500000032 2016 M13 11.15 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000032 55 55000000 32 U Average hourly earnings of production and nonsupervisory employees 5 CEU5500000034 2016 M13 111.6 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000034 55 55000000 34 U Indexes of aggregate weekly hours of production and nonsupervisory employees 6 CEU5500000035 2016 M13 179.2 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000035 55 55000000 35 U Indexes of aggregate weekly payrolls of production and nonsupervisory employees 7 CEU5500000081 2016 M13 236997 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000081 55 55000000 81 U Aggregate weekly hours of production and nonsupervisory employees 8 CEU5500000082 2016 M13 6189003 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000082 55 55000000 82 U Aggregate weekly payrolls of production and nonsupervisory employees 9 CEU5552200007 2016 M13 37.3 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5552200007 55 55522000 7 U Average weekly hours of production and nonsupervisory employees

2. <span style="font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">Join series and datatype on data_type_code</span>

In [17]:
SELECT TOP 10 *
FROM series AS s
LEFT JOIN datatype AS dt
ON s.data_type_code = dt.data_type_code

(10 rows affected)

Total execution time: 00:00:00.044

series_id supersector_code industry_code data_type_code seasonal series_title data_type_code data_type_text CES0000000001 0 00000000 1 S All employees 1 ALL EMPLOYEES CES0000000010 0 00000000 10 S Women employees 10 WOMEN EMPLOYEES CES0000000025 0 00000000 25 S All employees 25 ALL EMPLOYEES CES0000000026 0 00000000 26 S All employees 26 ALL EMPLOYEES CES0500000001 5 05000000 1 S All employees 1 ALL EMPLOYEES CES0500000002 5 05000000 2 S Average weekly hours of all employees 2 AVERAGE WEEKLY HOURS OF ALL EMPLOYEES CES0500000003 5 05000000 3 S Average hourly earnings of all employees 3 AVERAGE HOURLY EARNINGS OF ALL EMPLOYEES CES0500000006 5 05000000 6 S Production and nonsupervisory employees 6 PRODUCTION AND NONSUPERVISORY EMPLOYEES CES0500000007 5 05000000 7 S Average weekly hours of production and nonsupervisory employees 7 AVERAGE WEEKLY HOURS OF PRODUCTION AND NONSUPERVISORY EMPLOYEES CES0500000008 5 05000000 8 S Average hourly earnings of production and nonsupervisory employees 8 AVERAGE HOURLY EARNINGS OF PRODUCTION AND NONSUPERVISORY EMPLOYEES

3. <span style="font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">Join series and industry on industry_code</span>

In [18]:
SELECT TOP 10 *
FROM series AS s
JOIN industry AS i
ON s.industry_code = i.industry_code

(10 rows affected)

Total execution time: 00:00:00.050

series_id supersector_code industry_code data_type_code seasonal series_title id industry_code naics_code publishing_status industry_name display_level selectable sort_sequence CES0000000001 0 00000000 1 S All employees 0 0 - B Total nonfarm 0 T 1 CES0000000010 0 00000000 10 S Women employees 0 0 - B Total nonfarm 0 T 1 CES0000000025 0 00000000 25 S All employees 0 0 - B Total nonfarm 0 T 1 CES0000000026 0 00000000 26 S All employees 0 0 - B Total nonfarm 0 T 1 CES0500000001 5 05000000 1 S All employees 1 5000000 - A Total private 1 T 2 CES0500000002 5 05000000 2 S Average weekly hours of all employees 1 5000000 - A Total private 1 T 2 CES0500000003 5 05000000 3 S Average hourly earnings of all employees 1 5000000 - A Total private 1 T 2 CES0500000006 5 05000000 6 S Production and nonsupervisory employees 1 5000000 - A Total private 1 T 2 CES0500000007 5 05000000 7 S Average weekly hours of production and nonsupervisory employees 1 5000000 - A Total private 1 T 2 CES0500000008 5 05000000 8 S Average hourly earnings of production and nonsupervisory employees 1 5000000 - A Total private 1 T 2

## <span style="font-size: 12pt; font-family: Montserrat, sans-serif; color: rgb(53, 74, 95); font-weight: 700; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Aggregate Your Friends and Code some SQL</span>

<span style="font-size: 10pt; font-family: Arvo, serif; color: rgb(102, 102, 102); font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Put together the following:</span>

1. <span style="font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">How many employees were reported in 2016 in all industries? Round to the nearest whole number.&nbsp; No chopping people into little bits, please.</span>

<span style="font-size: 13.3333px; white-space: pre-wrap;"><b>There were 2,340,612 reported in 2016 for all industries.</b></span>

In [19]:
SELECT ROUND(SUM([value]),0) AS total_employees
FROM annual_2016
WHERE series_id IN (
    SELECT series_id
    FROM series
    WHERE data_type_code IN (
        SELECT data_type_code
        FROM datatype
        WHERE data_type_text = 'ALL EMPLOYEES'
    )
)

(1 row affected)

Total execution time: 00:00:00.067

total_employees 2340612

2. <span style="font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">How many women employees were reported in 2016 in all industries? Round to the nearest whole number.&nbsp;</span> 

**There were** <span style="font-size: 13.3333px; white-space: pre-wrap;"><b>1,125,490 women</b></span> **reported in 2016 for all industries.**<span style="font-size: 13.3333px; white-space: pre-wrap;"><br></span>

In [20]:
SELECT ROUND(SUM([value]),0) AS total_women_emp
FROM annual_2016
WHERE series_id IN (
    SELECT series_id
    FROM series
    WHERE data_type_code =10
)

(1 row affected)

Total execution time: 00:00:00.054

total_women_emp 1125490

3. <span style="font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">How many production/nonsupervisory employees were reported in 2016? Round to the nearest whole number.&nbsp;</span> 

<span style="font-size: 13.3333px; white-space: pre-wrap;"><b>There were 1,263,650 production/nonsupervisory employees were reported in 2016.</b></span>

In [23]:
SELECT ROUND(SUM([value]),0) AS 'production/nonsupervisory employees'
FROM annual_2016 AS a
JOIN series AS s
ON a.series_id = s.series_id
WHERE series_title = 'PRODUCTION AND NONSUPERVISORY EMPLOYEES'

(1 row affected)

Total execution time: 00:00:00.052

production/nonsupervisory employees 1263650

4. <span style="font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">In January 2017, what is the average weekly hours worked by production and nonsupervisory employees across all industries?</span>

Commands completed successfully.

Total execution time: 00:00:00

5. <span style="font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">What is the total weekly payroll for production and nonsupervisory employees across all industries in January 2017? Round to the nearest penny, please.</span>

In [11]:
USE LaborStatisticsDB

SELECT TOP 50*
FROM datatype
WHERE data_type_text LIKE '%weekly payroll%'

SELECT ROUND(SUM([value]), 2) AS total_wkly_payroll_prodandnonsupe 
FROM january_2017
WHERE series_id IN (
    SELECT series_id
    FROM series
    WHERE data_type_code = 82
)

(4 rows affected)

(1 row affected)

Total execution time: 00:00:00.089

data_type_code data_type_text 17 INDEXES OF AGGREGATE WEEKLY PAYROLLS OF ALL EMPLOYEES 35 INDEXES OF AGGREGATE WEEKLY PAYROLLS OF PRODUCTION AND NONSUPERVISORY EMPLOYEES 57 AGGREGATE WEEKLY PAYROLLS OF ALL EMPLOYEES 82 AGGREGATE WEEKLY PAYROLLS OF PRODUCTION AND NONSUPERVISORY EMPLOYEES

total_wkly_payroll_prodandnonsupe 1838753220

6. <span style="font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">In January 2017, which industry was the average weekly hours worked by production and nonsupervisory employees the highest? Which industry was the lowest?</span>

<span style="font-size: 13.3333px; white-space: pre-wrap;"><b>The average weekly hours worked by production and nonsupervisory employees across all industries were 36 hours.</b></span>

In [25]:
SELECT *
FROM datatype
WHERE data_type_text LIKE '%AVERAGE WEEKLY HOURS OF PRODUCTION AND NONSUPERVISORY EMPLOYEES%'


SELECT AVG([value])
FROM dbo.january_2017
JOIN dbo.series
ON january_2017.series_id = series.series_id
WHERE data_type_code = 7

(1 row affected)

(1 row affected)

Total execution time: 00:00:00.074

data_type_code data_type_text 7 AVERAGE WEEKLY HOURS OF PRODUCTION AND NONSUPERVISORY EMPLOYEES

(No column name) 36.05862068965521

7. <span style="font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">In January 2021, which industry was the total weekly payroll for production and nonsupervisory employees the highest? Which industry was the lowest?</span>

<span style="font-size: 13.3333px; white-space: pre-wrap;"><b>Highest weekly payroll: Total private.</b></span>

<span style="font-size: 13.3333px; white-space: pre-wrap;"><b>Lowest weekly payroll: Coin-operated laundries and drycleaners</b></span>

In [26]:
SELECT TOP 1 s.industry_code, SUM(j.value) AS total_wkly_payroll, i.industry_name
FROM january_2017 as j 
LEFT JOIN series as s 
ON j.series_id = s.series_id
--WHERE s.data_type_code = 7
LEFT JOIN industry as i 
ON s.industry_code = i.industry_code
WHERE s.data_type_code = 82
GROUP BY s.industry_code, i.industry_name
ORDER BY SUM(j.[value]) DESC, s.industry_code, i.industry_name 

SELECT TOP 1 s.industry_code, SUM(j.value) AS total_wkly_payroll, i.industry_name
FROM january_2017 as j 
LEFT JOIN series as s 
ON j.series_id = s.series_id
--WHERE s.data_type_code = 7
LEFT JOIN industry as i 
ON s.industry_code = i.industry_code
WHERE s.data_type_code = 82
GROUP BY s.industry_code, i.industry_name
ORDER BY SUM(j.[value]) ASC, s.industry_code, i.industry_name 
     

(1 row affected)

(1 row affected)

Total execution time: 00:00:00.119

industry_code total_wkly_payroll industry_name 05000000 295944946 Total private

industry_code total_wkly_payroll industry_name 80812310 40448 Coin-operated laundries and drycleaners

## <span style="font-size: 12pt; font-family: Montserrat, sans-serif; color: rgb(53, 74, 95); font-weight: 700; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Subqueries, Unions, Derived Tables, Oh My!</span>

1. <span style="font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">Write a query that returns the series_id, industry_code, industry_name, and value from the january_2017 table but only if that value is greater than the average value for annual_2016 of data_type_code 82.</span>

In [28]:
SELECT TOP 25 j.series_id, i.industry_code, i.industry_name, j.[value]
FROM january_2017 AS j, industry AS i, series AS s
WHERE j.series_id = s.series_id AND s.industry_code = i.industry_code
AND s.data_type_code= 82 AND j.[value] > (
    SELECT AVG(annual_2016.[value])
    FROM annual_2016
    WHERE annual_2016.series_id IN (
        SELECT series_id
        FROM series
        WHERE data_type_code = 82
    )
)
GROUP BY j.series_id, i.industry_code, i.industry_name, j.[value]


(25 rows affected)

Total execution time: 00:00:00.088

series_id industry_code industry_name value CES5500000082 55000000 Financial activities 6328968 CES5552200082 55522000 Credit intermediation and related activities 1717912 CES5552210082 55522100 Depository credit intermediation 971796 CES5552300082 55523000 NULL 1044224 CES5552400082 55524000 Insurance carriers and related activities 2347328 CES5552410082 55524100 Insurance carriers 1477799 CES5552420082 55524200 Insurance agencies 872398 CES5553100082 55531000 Real estate 862987 CEU5500000082 55000000 Financial activities 6467426 CEU5552200082 55522000 Credit intermediation and related activities 1767137 CEU5552210082 55522100 Depository credit intermediation 1004069 CEU5552300082 55523000 NULL 1087325 CEU5552400082 55524000 Insurance carriers and related activities 2408490 CEU5552410082 55524100 Insurance carriers 1509979 CEU5552420082 55524200 Insurance agencies 894878 CEU5553100082 55531000 Real estate 865620 CES3100000082 31000000 Durable goods 4859716 CES3133200082 31332000 Fabricated metal products 871298 CES3133600082 31336000 Transportation equipment 1257764 CEU3100000082 31000000 Durable goods 4808368 CEU3133200082 31332000 Fabricated metal products 867242 CEU3133600082 31336000 Transportation equipment 1242685 CES6500000082 65000000 Education and health services 14765156 CES6562000082 65620000 Health care and social assistance 12762526 CES6562000182 65620001 NULL 11413008

2. <span style="font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">Create a Union table comparing average weekly earnings&nbsp; of production and nonsupervisory employees between annual_16 and january_17 using the data type 30.&nbsp; Round to the nearest penny.&nbsp; You should have a column for the average earnings and a column for the year, and the period.</span>

In [29]:
SELECT 'Annual_16' AS source, ROUND(avg(a.[value]), 2) AS avg_weekly_earnings, a.[year] AS results_year, 
a.[period] AS results_period
FROM annual_2016 AS a, series AS s
WHERE a.series_id = s.series_id 
AND s.data_type_code = 30
GROUP BY a.[year], a.[period]

UNION
SELECT 'January_17', ROUND(AVG(j.[value]), 2), j.[year], j.[period]
FROM january_2017 AS j, series AS s 
WHERE j.series_id = s.series_id 
AND s.data_type_code = 30
GROUP BY j.[year], j.[period]

(2 rows affected)

Total execution time: 00:00:00.250

source avg_weekly_earnings results_year results_period Annual_16 797.2 2016 M13 January_17 808.53 2017 M01

## <span style="font-size:12pt;font-family:Montserrat, sans-serif;color:rgb(53, 74, 95);font-weight:700;vertical-align:baseline;white-space:pre-wrap;">Summarize Your Results</span>

<span style="font-size:10pt;font-family:Arvo, serif;color:rgb(102, 102, 102);vertical-align:baseline;white-space:pre-wrap;">With what you know now about the </span> <span style="color:rgb(102, 102, 102);font-family:&quot;Times New Roman&quot;;font-size:13.3333px;white-space:pre-wrap;">Bureau of Labor Statistics (BLS) Current Employment Survey (CES) results and working with the</span> <span style="color:rgb(102, 102, 102);white-space:pre-wrap;font-family:&quot;Times New Roman&quot;;font-size:13.3333px;">Labor Statistics Database</span><span style="color:rgb(102, 102, 102);font-family:Arvo, serif;font-size:10pt;white-space:pre-wrap;">, answer the following questions. Note that while this is subjective, you should include relevant data to back up your opinion.</span>

1. <span style="font-size:10pt;white-space:pre-wrap;color:rgb(102, 102, 102);font-family:Arvo, serif;">During which time period did production and nonsupervisory employees fare better?</span>

Production and Nonsupervisory employees fared better during January of 2017 opposed to 2016. The average weekly earnings of the empolyees during 2016 was $707.20 whereas the earnings were $808.53 for January 2017.

2. <span style="font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">In which industries did production and nonsupervisory employees fare better?</span>

The "Total Private" Industry Employees fared better than other fields along with "<span style="font-size: 14px;">Private service-providing" and "Professional and business services".&nbsp;&nbsp;The lowest performing industry based on total weekly payroll was Coin-Operated Laundries and Dry Cleaners</span>

3. <span style="font-size: 10pt; font-family: Arvo, serif; color: rgb(102, 102, 102); font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Now that you have explored the datasets, is there any data or information that you wish you had in this analysis?</span>

I think it would be interesting to compare data from at least 2 complete years. Especially if there were signicant changes in national unemployment rates or if the nation was in a recession. Analysing the data based result of econic changes within specific industries could be very intriguing and benefitical for various companies.